app/scripts/lib/strings.js

```diff
@@ -5,10 +5,14 @@
 define(function (require, exports, module) {
   'use strict';
 
+  const t = (msg) => msg;
 
-  var t = function (msg) {
-    return msg;
-  };
+  const HTML_CHAR_CODE_REGEXP = /&\D+;/i;
+  const HTML_TAG_REGEXP = /<.*>/;
+  const NAMED_VARIABLE_REGEXP = /\%\(([a-zA-Z]+)\)s/g;
+  const UNNAMED_VARIABLE_REGEXP = /%s/g;
+  // safe variables have an `escaped` prefix.
+  const UNSAFE_VARIABLE_REGEXP = /\%\(((?!escaped)[a-zA-Z]+)\)s/g;
```

Comment:
We've been moving away from a postfix-Hungarian notation for RegExps in the auth server. I don't know if that's desirable here as well.